network-store-iidm-tck/src/test/java/com/powsybl/network/store/tck/ActivePowerLimitsIT.java

```diff
@@ -23,4 +24,7 @@
 @TestPropertySource(properties = { "spring.config.location=classpath:application.yaml" })
 public class ActivePowerLimitsIT extends AbstractActivePowerLimitsTest {
 
+    @Test
+    public void avoidInvalidTestClassError() { }
```

Comment:
If you run a test class without any test in it you get a "invalidTestClass exception" (even if you extend a class containing tests) so I added these to avoid that issue